chai-dom.js

```diff
@@ -120,17 +120,22 @@
     flag(this, 'trim-text', true)
   })
 
+  chai.Assertion.addChainableMethod('rendered', null, function() {
```

Comment:
This should use `addProperty` and not `addChainableMethod`, as there's no support for this being called as a function/method.